src/CoreShop/Bundle/FrontendBundle/Twig/LocaleSwitcherExtension.php

```diff
@@ -35,6 +37,7 @@ public function __construct(
         private Document\Service $documentService,
         private ShopperContextInterface $shopperContext,
         private RequestStack $requestStack,
+        protected ContainerInterface $container,
```

Comment:
never ever inject the whole container! inject what you need, in this case RouterInterface.

Reply:
@BlackbitDevs I made the change now. We are preparing the release today

Reply:
Thanks @dpfaffenbauer